### PR TITLE
Adding UDFs for two value comparisons of big numbers

### PR DESCRIPTION
**Pull request by lingzhong**

### CODE REVIEW

File: udfs/community/README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ SELECT bqutil.fn.int(1.684)
 * [bignumber_add](#bignumber_add-first-string-second-string)
 * [bignumber_avg](#bignumber_avg-numbers-array)
 * [bignumber_div](#bignumber_div-first-string-second-string)
+* [bignumber_gt](#bignumber_gt-first-string-second-string)
```
Comment (danieldeleo) [inline udfs/community/README.md:19]: Please add an example of usage in README, thanks!

File: udfs/community/README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ SELECT bqutil.fn.int(1.684)
...
 * [bignumber_mul](#bignumber_mul-first-string-second-string)
 * [bignumber_sub](#bignumber_sub-first-string-second-string)
 * [bignumber_sum](#bignumber_sum-numbers-array)
```
Comment (danieldeleo) [inline udfs/community/README.md:22]:
```suggestion
* [bignumber_add](#bignumber_addfirst-string-second-string)
* [bignumber_avg](#bignumber_avgnumbers-array)
* [bignumber_div](#bignumber_divfirst-string-second-string)
* [bignumber_gt](#bignumber_gtfirst-string-second-string)
* [bignumber_mul](#bignumber_mulfirst-string-second-string)
* [bignumber_sub](#bignumber_subfirst-string-second-string)
* [bignumber_sum](#bignumber_sumnumbers-array)
```

File: udfs/community/README.md [hunk 3]
```diff
@@ -13,12 +13,13 @@ SELECT bqutil.fn.int(1.684)
...
-* [bignumber_add](#bignumber_add-first-string-second-string)
-* [bignumber_avg](#bignumber_avg-numbers-array)
-* [bignumber_div](#bignumber_div-first-string-second-string)
-* [bignumber_mul](#bignumber_mul-first-string-second-string)
-* [bignumber_sub](#bignumber_sub-first-string-second-string)
-* [bignumber_sum](#bignumber_sum-numbers-array)
+* [bignumber_add](#bignumber_addfirst-string-second-string)
+* [bignumber_avg](#bignumber_avgnumbers-array)
+* [bignumber_div](#bignumber_divfirst-string-second-string)
+* [bignumber_gt](#bignumber_gtfirst-string-second-string)
```
Comment (danieldeleo) [inline udfs/community/README.md:19]: Please add example usage in README, thanks!

Comment (lingzhong) [reply udfs/community/README.md:19]: Done.

**Comment by lingzhong** [comment]
@danieldeleo if `gt` looks good, then I'll add code for `lt`, `gte`, `lte`, `eq` to complete the comparison suite. Thanks!

**Review by danieldeleo — APPROVED** [review]

**Comment by danieldeleo** [comment]
> @danieldeleo if `gt` looks good, then I'll add code for `lt`, `gte`, `lte`, `eq` to complete the comparison suite. Thanks!

looks good! 👍

**Comment by danieldeleo** [comment]
@lingzhong are you going to add them to this PR?

**Comment by lingzhong** [comment]
> @lingzhong are you going to add them to this PR?

Yep! Added. Please see https://github.com/GoogleCloudPlatform/bigquery-utils/pull/349/commits/de21d95b7ab1b60fbf30968e82f7049431e42e30

**Pull request merged**